test/tsconfig.json

```diff
@@ -0,0 +1,18 @@
+{
+	"extends": "../tsconfig.json",
+	"compilerOptions": {
+		"moduleResolution": "node",
+		"paths": {
+			"chai": [
+				"./plugins/oas"
```

Comment:
there is no such plugin in the codebase, is that required?

Reply:
No, wrongly copied from the other repo.